get-dependency-list.js

```diff
@@ -22,12 +24,45 @@ module.exports = function(filename, serverless) {
   const modulesToProcess = [];
   const localFilesToProcess = [filename];
 
+  const shouldUseLocalNodeModules = lodash.get(serverless.service.custom, 'serverless-plugin-include-dependencies.shouldUseLocalNodeModules', false);
```

Comment:
Is there a chance serverless will be undefined?

Reply:
No, and if so - pipeline will fall before it will get to this plugin